python/ray/_private/usage/usage_lib.py

```diff
@@ -66,6 +66,23 @@
 #################
 
 
+@dataclass(init=True)
+class ClusterConfigToReport:
```

Comment:
nit: 

Would `ClusterConfigReport` be a better name for this class?
The class itself isn't doing any conversion so `To` doesn't seem to belong.

Same below: `ClusterStatusReport`

Reply:
I'm trying to be consistent with other dataclass there: `UsageStatsToReport` and `UsageStatsToWrite`

Reply:
Sgtm